openapi/components/schemas/QuoteCreateOrder.yaml

```
@@ -0,0 +1,467 @@
+type: object
+title: QuoteCreateOrder
+required:
+  - websiteId
+  - customerId
+  - items
+  - action
+properties:
+  id:
+    readOnly: true
+    description: ID of the quote.
+    type: string
+    maxLength: 50
+    example: qt_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  type:
+    description: Specifies the type of the quote. A quote can be a subscription or a one-time purchase.
+    type: string
+    readOnly: true
+    enum:
+      - subscription-order
+      - one-time-order
+  action:
+    description: |-
+      Action of the quote.
+      When a quote is accepted, a new order is created. `orderId` is filled with new generated value from the order.
```

suggestion: for readability 
```suggestion
      When a quote is accepted, a new order is created. 
      The `orderId` value is filled with a new generated value from the order.
```